WBooks/Screens/BookLibrary/BookLibraryController.swift

```diff
@@ -22,7 +22,7 @@ class BookLibraryController: UIViewController {
         fatalError("init(nibName:bundle:) has not been implemented")
     }
     
-    lazy var libraryViewModel: BookLibraryViewModel = {
+    var libraryViewModel: BookLibraryViewModel = {
```

Comment:
it doesn't need to be a computed property. `var libraryViewModel = BoolLibraryViewModel()`